train.py

```diff
@@ -481,12 +494,37 @@ def main(args):
 			onnxruntime.set_default_logger_severity(0)
 		(logits_, ) = onnxruntime_session.run(None, dict(x = waveform_input.cpu().numpy()))
 
-		assert torch.allclose(
+		wraper = models.OnnxWrapper(args.onnx)
```

Comment:
when you're fixing a typo, please do the search everywhere. this typo was already fixed in the class name

Reply:
done